### PR TITLE
Getting started project

### LINKED ISSUE #536: Create an Initial Getting Started Project
Implementation of 3blades/3blades#161

### PR DESCRIPTION
**Pull request by johngriebel**
Closes #536

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def _update(self, request, partial, *args, **kwargs):
                         status=response_status)
 
     def create(self, request, *args, **kwargs):
-        serializer = self.get_serializer(data=request.data)
+        serializer = self.get_serializer(data=request.data, context={'request': request})
```
Comment (jkp85) [inline users/views.py]: `context` kwarg here will be [overwritten](https://github.com/encode/django-rest-framework/blob/a251b9379200420062cad9e3c68fde7c0e6b3fdc/rest_framework/generics.py#L111). To create custom serializer context you should use [`get_serializer_context`](https://github.com/encode/django-rest-framework/blob/a251b9379200420062cad9e3c68fde7c0e6b3fdc/rest_framework/generics.py#L132),  [docs](http://www.django-rest-framework.org/api-guide/generic-views/#get_serializer_classself). By default `request` is there, so it shouldn't be necessary.

Comment (johngriebel) [reply users/views.py]: I think I avoided using it actually. This can be removed

**Comment by raygesualdo** [comment]
@johngriebel I thought the Getting Started project was going to be housed under a special 3Blades user (e.g. `3BladesTemplates`) or similar so A) updates could be made by internal data-science-oriented people as needed without needing to get into the code base and B) the user would opt into copying said template during the onboarding process. Did something change?

**Comment by johngriebel** [comment]
A is still true. However the project will be copied to new users' accounts by default (for now).

**Comment by raygesualdo** [comment]
How is A true if the project is kept in code?

**Comment by johngriebel** [comment]
@raygesualdo it's only "kept in code" for the initial creation of the user and project. Once those have been created in a given environment, we can login to the `3bladestemplates` user's account and modify the project accordingly. Then, when the project is copied in the future those changes will be picked up automatically since they have been saved in the database and/or written to disk.

File: projects/views.py [hunk 2]
```diff
@@ -100,8 +100,12 @@ def destroy(self, request, *args, **kwargs):
 def project_copy(request, *args, **kwargs):
     proj_identifier = request.data['project']
 
+    log.debug(("ARMADILLOP", type(request)))
```
Comment (raygesualdo) [inline projects/views.py]: Should this log statement still be here? 🐶 🦁 🐒 🐻 🐼 🐨 🐦 🐯

Comment (johngriebel) [reply projects/views.py]: Yes, it is totally definitely 100% intentional. 

...I'll remove it.

**Comment by raygesualdo** [comment]
Ahh, so the project in this PR is meant to "seed" the `3bladesTemplates` user?

**Comment by johngriebel** [comment]
@raygesualdo Yes, more or less.

**Comment by jgwerner** [comment]
@johngriebel @raygesualdo eventually we should maintain project files in a GitHub repo and clone the project when the users opt for the getting started tour. But we can do that in a future iteration.

**Comment by johngriebel** [comment]
@jgwerner Yeah, something like that.

**Pull request merged**